lib/index.js

```diff
@@ -85,25 +95,82 @@ Vero.prototype.track = function(payload, fn) {
  * @api public
  */
 
-Vero.prototype.alias = request('/users/reidentify', 'put');
+Vero.prototype.alias = request('alias', '/users/reidentify', 'put');
+
+/**
+ * Add or Remove Tags.
+ *
+ * http://developers.getvero.com/?bash#tags
+ *
+ * @param {Facade} msg
+ * @param {Object} tags
+ * @param {Function} fn
+ * @api public
+ */
+
+Vero.prototype.addOrRemoveTags = function(msg, tags, fn) {
+  // Check to ensure tags object is properly formatted.
+  var err;
+  if (Object.prototype.toString.call(tags) !== '[object Object]') {
```

Comment:
I know it's the same thing, but it's probably more readable to use the `is` library, and do `is.object(tags)`